spec/index.html

```diff
@@ -0,0 +1,634 @@
+        companyURL: "https://www.mi.com/",
+        company: "Xiaomi",
+        w3cid: 124040
+      }],
+
+      group: "cg/miniapps",
+      github: {
+        repoURL: "https://github.com/w3c/miniapp-widget",
+        branch: "main",
+      },
+    };
+  </script>
+</head>
+
+<body>
+  <section id='abstract'>
+    <p its-locale-filter-list="en" lang="en">
+      The MiniApp Widget is a special form of <a href="https://www.w3.org/TR/miniapp-packaging/#dfn-page">MiniApp Page</a>.
+        Unlike a page, a widget can occupy a certain area instead of the entire screen.
+        It is used to display key information and respond to simple user operations,
+        such as displaying the weather at the user’s current location, or tracking the
+        user’s current travel status and providing further actions.
+    </p>
+    <p its-locale-filter-list="zh-hans" lang="zh-hans">
+      MiniApp Widget是<a href="https://www.w3.org/TR/miniapp-packaging/#dfn-page">MiniApp 页面</a>的一种特殊形式。
+      与页面不同，widget可以仅占用屏幕的部分区域（而不是全部），用于为用户显示关键信息和响应简单的用户操作，例如显示用户所在地的天气，跟踪用户当前的行程状态并提供进一步的操作。
+    </p>
+    <p its-locale-filter-list="en" lang="en">
+      This specification defines addressing, packaging, manifest and lifecycle for a MiniApp Widget. The interaction and communication between MiniApp widget and MiniApp or host environment are also defined.
+    </p>
+    <p its-locale-filter-list="zh-hans" lang="zh-hans">
+      本标准规定了MiniApp Widget的addressing、packaging、manifest和lifecycle要求，还规定了MiniApp widget与MiniApp或与宿主环境间进行互动交流的要求。
+    </p>
+  </section>
+
+  <section id='sotd'>
+    <p id="langSwitch">
+      <button onclick="switchLang('zh-hans')" lang="zh-hans">简体中文</button>
+      <button onclick="switchLang('en')" lang="en">English</button>
+      <button onclick="switchLang('all')" lang="en">All</button>
+    </p>
+  </section>
+
+  <section id='introduction'>
+    <h2>
+      <span its-locale-filter-list="en" lang="en">Introduction</span>
+      <span its-locale-filter-list="zh-hans" lang="zh-hans">介绍</span>
+    </h2>
+
+    <section id='overview'>
+      <h3>
+      <span its-locale-filter-list="en" lang="en">Overview</span>
+      <span its-locale-filter-list="zh-hans" lang="zh-hans">概述</span>
+      </h3>
+      <p its-locale-filter-list="zh-hans" lang="zh-hans">MiniApp widget 是 MiniApp页面的一种特殊形式。与页面不同，widget可以仅占用屏幕的部分区域（而不是全部），
+        用于为用户显示关键信息和响应简单的用户操作，例如显示用户所在地的天气，跟踪用户当前的行程状态并提供进一步的操作。</p>
+  
+      <p its-locale-filter-list="en" lang="en">MiniApp widget is a special form of MiniApp page. Unlike pages, widgets can use only part of the screen (rather than all of
+      it) and are used to display key information to the user and respond to simple
+      user actions, such as displaying the weather at the user's location, tracking
+      user's current travel status and providing next actions .</p>
+    </section>
+
+    <section id='terminology'>
+      <h3>
+      <span its-locale-filter-list="en" lang="en">Terminology</span>
```

Comment:
I would suggest that we link to https://www.w3.org/TR/miniapp-packaging/#sec-terminology directly instead of defining the same terms here. Otherwise, we are maintaining two glossaries and synchronizing the definitions on both sides.